includes/class-evf-smart-tags.php

```diff
@@ -88,6 +88,62 @@ public function process( $content, $form_data, $fields = '', $entry_id = '' ) {
 							);
 						}
 					}
+
+					// Properly display Radio field in smart tag.
+					if ( isset( $value['image'] ) && 'radio' === $fields[ $mixed_field_id[1] ]['type'] ) {
+						if ( ! is_array( $value ) && false !== strpos( $value['image'], $uploads['basedir'] ) ) {
+							$value = trailingslashit( content_url() ) . str_replace( str_replace( 'uploads', '', $uploads['basedir'] ), '', $value['image'] );
+						}
+
+						if ( ! empty( $value ) ) {
+							$value = sprintf(
+								'<br> <img src="%s" style="width:150px;height:80px;max-height:200px;max-width:100px;"/> <br>%s',
```

Comment:
@JackRourkeK I am really not a fan of `<br>` tags but can we simplify this and prevent some whitespace. Also for others below :)

Reply:
Resolved and updated with line break when multiple image and files are uploaded.